platform/viewer/src/connectedComponents/ToolbarRow.js

```diff
@@ -308,9 +309,17 @@ function _handleBuiltIn(button) {
   }
 
   if (options.behavior === 'DOWNLOAD_SCREEN_SHOT') {
+    const activeViewportIndex = commandsManager.runCommand(
+      'getActiveViewportIndex'
+    );
```

Comment:
If you shift `modal.show` to inside a command, you can get rid of the `getActiveViewportIndex` command.

Reply:
Done! thx.